src/routes/versions.rs

```diff
@@ -703,15 +711,7 @@ pub async fn delete_file(
     .map_err(|e| ApiError::DatabaseError(e.into()))?;
 
     if let Some(row) = result {
-        let version_data = database::models::Version::get_full(
-            database::models::VersionId(row.version_id),
-            &**pool,
-        )
-        .await
-        .map_err(|e| ApiError::DatabaseError(e.into()))?;
-
-        if let Some(data) = version_data {
-            let mut transaction = pool
+         let mut transaction = pool
```

Comment:
1. The indentation is messed up past this point, run cargo fmt and fix the sql query strings.
2. Wasn't this verifying permissions somehow? How is it doing that now?

Reply:
It still checks them in the query